src/styles/blocks/stars--0.css

```diff
@@ -0,0 +1,44 @@
+.stars--0 {
+  display: flex;
+  margin-top: 22px;
+}
+
+.stars--0 .stars__star:nth-child(1) {
+  display: block;
+  background-image: url(../../images/star.svg);
+  width: 13px;
+  height: 12.2px;
+  margin-left: 22px;
```

Comment:
Avoid using fixed margins like `margin-left: 22px;` for the first star. If you need to space the stars from the container, consider adding padding to the container instead. This will make it easier to manage the layout if the number of stars or their placement changes.